htdocs/product/price.php

```diff
@@ -874,16 +875,21 @@ function on_change() {
 		print '<tr><td>';
 		$text = $langs->trans('MinPrice');
 		print $form->textwithpicto($text, $langs->trans("PrecisionUnitIsLimitedToXDecimals", $conf->global->MAIN_MAX_DECIMALS_UNIT), 1, 1);
+		print '</td><td';
+		if (empty($conf->global->PRODUCT_MINIMUM_RECOMMENDED_PRICE)) {
+			print ' colspan="2"';
+		}
+		print '>';
 		if ($object->price_base_type == 'TTC') {
-			print '<td><input name="price_min" size="10" value="' . price($object->price_min_ttc) . '">';
+			print '<input name="price_min" size="10" value="' . price($object->price_min_ttc) . '">';
 		} else {
-			print '<td><input name="price_min" size="10" value="' . price($object->price_min) . '">';
+			print '<input name="price_min" size="10" value="' . price($object->price_min) . '">';
 		}
 		if ( !empty($conf->global->PRODUCT_MINIMUM_RECOMMENDED_PRICE))
 		{
-			print '<td align="left">'.$langs->trans("MinimumRecommendedPrice", price($maxpricesupplier,0,'',1,-1,-1,'auto')).' '.img_warning().'</td>';
+			print '</td><td align="left">'.$langs->trans("MinimumRecommendedPrice", price($maxpricesupplier,0,'',1,-1,-1,'auto')).' '.img_warning().'</td>';
 		}
-		print '</td></tr>';
+		print '</tr>';
```

Comment:
The closing td is included in the previous if, but if the if is false, no closing td is done